src/Service.Tests/dab-config.MsSql.json

```diff
@@ -10,7 +10,8 @@
   "runtime": {
     "rest": {
       "enabled": true,
-      "path": "/api"
+      "path": "/api",
+      "request-body-strict": true
```

Comment:
Im guessing these config files became stale and the pipeline uses the files generated by CLI. Could we add a check to compare the files generated by CLI and these checked in files so we catch the staleness when the CLI commands are changed itself?

Reply:
Ack. I'll add a test to compare the checked-in sample config file and the one generated for integration tests

Reply:
The reference config files need to be renamed as the integration tests also generate the config files with the same name (The script deletes any existing files with the same name). So, to compare the contents of the CLI generated config file vs check-in config file, both should exist at the same time.

I started implementing tests by renaming these reference config files and realized that the absence of the `dab-config*.json` files in the Tests directory leads to build failure. To implement this validation some more changes are necessary but they are not related to the objective of this PR.  So, I'll implement these validations as a separate PR.

I've created [this](https://github.com/Azure/data-api-builder/issues/1766) issue to track this

Reply:
I've opened [this PR](https://github.com/Azure/data-api-builder/pull/1768) for this validation